modules/osdk-ha-sno-api.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-ha-sno.adoc
+
+[id="osdk-ha-sno-api_{context}"]
+= Cluster high-availability mode API usage in Operators
+
+{product-title} provides a cluster high-availability mode API that can be used by Operators to help detect infrastructure topology. The Infrastructure API holds cluster-wide information regarding infrastructure. Operators managed by Operator Lifecycle Manager (OLM) can use the Infrastructure API if they need to configure an Operand or managed workload differently based on the high-availability mode.
+
+In the Infrastructure API, the `infrastructureTopology` status expresses the expectations for infrastructure services that do not run on control plane nodes, usually indicated by a node selector for a `role` value other than `master`. The `controlPlaneTopology` status expresses the expectations for Operands that normally run on control plane nodes.
+
+The default setting for either status is `HighlyAvailable`, which represents the behavior Operators have in default, multiple node clusters. The `SingleReplica` setting is used in single node clusters, also known as Single Node OpenShift (SNO), and indicates that Operators should not configure their Operands for high-availability operation.
```

Comment:
suggest moving the which clause to the end of the sentence instead of in the middle.

Reply:
Oh I think the comma in "default, multiple" is making it look like the which clause is in the middle. I'll drop the "default" there since it's fairly redundant at that point.